ca_gazebo/models/camera_world/camera/model.sdf

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='camera'>
+  <static>1</static>
+  <link name='camera_link'>
+    <visual name='visual'>
+      <geometry>
+        <box>
+          <size>0.1 0.1 0.1</size>
+        </box>
+      </geometry>
+    </visual>
+    <sensor type="camera" name="camera1">
+      <update_rate>30.0</update_rate>
+      <camera name="head">
+        <horizontal_fov>1.3962634</horizontal_fov>
+        <image>
+          <width>800</width>
+          <height>800</height>
+          <format>R8G8B8</format>
+        </image>
+        <clip>
+          <near>0.02</near>
+          <far>300</far>
+        </clip>
+        <noise>
+          <type>gaussian</type>
+          <mean>0.0</mean>
+          <stddev>0.007</stddev>
+        </noise>
+      </camera>
+      <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
+        <alwaysOn>true</alwaysOn>
```

Comment:
Is `alwaysOn` a valid plugin tag?

Reply:
Check `updateRate` too

Reply:
check [here](http://sdformat.org/spec?ver=1.6&elem=sensor#sensor_plugin)

Reply:
To what I saw, `always_on` is a tag for the sensor and `alwaysOn` is used in plugins, same case with updateRate